.github/workflows/android_release.yml

```
@@ -0,0 +1,30 @@
+name: Android Release
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)